xml/dm_registers.xml

```diff
@@ -120,6 +120,13 @@
         This register controls the overall debug module
         as well as the currently selected harts, as defined in \Fhasel.
 
+        {\tt HARTSELBITS} (the size of \Fhartsel) is chosen by an
+        implementation depending on its needs.  It must be at least 0 and at
+        most 10. The maximum number of harts a DM can support is $2^{\tt
+        HARTSELBITS}$. A debugger can discover {\tt HARTSELBITS} by writing all
+        ones to \Fhartsel (assuming the maximum size) and reading back the
+        value to see which bits were actually set.
+
```

Comment:
We may want to clarify a few points:

1) "can support" should perhaps be "can select", or "can give status on". It's unclear that this is not the same thing as how many harts are *actually* existent in the system.
2) Writing a hartsel value greater than 2^HARTSELBITS can result in aliasing to a hart that is supported, which may have unintended effects. So an implementation SHOULD determine HARTSELBITS as part of the probing sequence.

Reply:
1. How about "can control?"
2. Done.